### ISSUE TITLE
Allow for both `failOnAccessibilityError` and `baseline` flags to work at the same time

### ISSUE THREAD

**Issue by astherath** [microsoft/accessibility-insights-action#1892]
## Is your feature request related to a problem? Please describe.
The framework my team uses (FluentUI/Elixir) has autogenerated classnames for css and these are making the baseline file be different on every pass, therefore blocking PRs even though there were no new bugs introduced. 

## Describe the desired outcome
I would expect that if the `failOnAccessibilityError` flag is set to false then even if the baseline file is changed from the scan output, it would not fail the PR. This would allow my team (IcM) to track regressions with baseline files but not have them block PRs.

## Describe alternatives you've considered
I looked into making the css selectors/classnames be deterministic so that the baseline file wouldnt have false flags but this was not possible sadly.

## Additional context
Related issue from original question: #1815

**Comment by DaveTryon** [comment]
HI, @astherath! We discussed this in today's triage meeting. Feature #1802 already tracks the normalization that you describe, and is the way forward in the long term. Please feel free to upvote it so we have that captured in our prioritization conversations. 

For the short term, we decided not to change how the flags work, as it would be a breaking change to other users. There are a couple of workarounds that you might consider:
1. If baselines aren't useful, you can just turn them off
2. If you're getting some value from baselines but the error is a problem, you can modify your ADO pipeline to use the `continueOnError` option to ignore the failure, but then every subsequent step will need the `succeededOrFailed` condition. We do this in our validation pipeline [here](https://github.com/microsoft/accessibility-insights-action/blob/main/pipelines/ado-extension-validation-template.yaml#L65-L66).
